common/src/sql/dbinit.sql

```diff
@@ -365,6 +365,12 @@ CREATE UNIQUE INDEX ON omicron.public.instance (
 --     'faulted'
 -- );
 
+CREATE TYPE omicron.public.block_size AS ENUM (
+  'traditional',
+  'iso',
+  'advancedformat'
+);
+
```

Comment:
**[not blocking]**

Instead of carrying over these labels could we just do the byte value of the sizes in string form?

```suggestion
CREATE TYPE omicron.public.block_size AS ENUM (
  '512',
  '2048',
  '4096'
);
```

The labels as are don't seem to have as much meaning in context.

Reply:
nice, done in d5db0b5e